### ISSUE TITLE
mapImage() different colours if fillContour=TRUE

### ISSUE THREAD

**Issue by dankelley** [dankelley/oce#726]
I noticed this problem in #721. The image below is from 726a.R on the ``issue721`` branch of oce. Note the colour differences at the bottom portion of AA, with green on left panel and light blue on the right panel.  The bottom-left panel is depth along the crosses in the top-right panel -- notice that the green is occuring for the top-left panel at z=0 values.

Therefore, I suspect the top two panels differ because of a ``< ``versus ``<=`` problem in the code. Note that difficulties on colour transitions have come up over and over. **The question is -- does z=0 code to a land colour or a water one?**  Comments on this, @richardsc?

![726a_1](https://cloud.githubusercontent.com/assets/99469/9561927/b869d75e-4e31-11e5-9cc1-76c8b67df834.png)

**Comment by dankelley** [comment]
I updated to make the RH panel look like the LH one, but now LL panel of 655b.R (for #655) is broken again so I'm going to reopen that.

**Comment by dankelley** [comment]
I now judge the RH panel as shown above to be correct, so I reversed the code change indicate in the previous comment. That solves the reopened #655 so I closed that.  I also ran ``make mapImage`` and looked at all the graphs, and judge that at the moment (branch ``issue721``, commit a6216ea95e677c2fb94d29dea02f714d323c9862) we now again have a working ``mapImage()``. My next step is to make the LH panel above look like the RH panel.

**Comment by richardsc** [comment]
I'm late to this, but my opinion is that whatever `mapImage()` is doing with `filledContour=FALSE` in the RH plot is correct, and so `mapImage(..., filledContour=TRUE)` should do the same.

It's not clear to me without looking deeply at the code whether this means that `z=0` codes to either a "land" or "ocean" colour.

**Comment by dankelley** [comment]
Good, so we are in agreement. I've updated the code (you and I are working at the same time today) and the results now look as below. I'm closing this, because it seems settled.

NOTE: I've done a lot of messing about with issues, so that now I think ``make mapImage`` shows things are working well. Some of my work has uncovered problems with the longitude conversion that we had recently, and I've fixed them too. For this reason, the R code for something like 10 or so issues has been tweaked lately.

![726a](https://cloud.githubusercontent.com/assets/99469/9567672/5b8dddf8-4f0a-11e5-8118-2ffab6efcc74.png)
